neuralprophet/configure.py

```diff
@@ -500,18 +500,58 @@ class Event:
 
 ConfigEvents = OrderedDictType[str, Event]
 
-
 @dataclass
 class Holidays:
-    country: Union[str, List[str]]
+    country: Union[str, List[str], dict]
     lower_window: int
     upper_window: int
     mode: str = "additive"
     reg_lambda: Optional[float] = None
     holiday_names: set = field(init=False)
 
     def init_holidays(self, df=None):
-        self.holiday_names = utils.get_holidays_from_country(self.country, df)
+        self.holiday_names = self.get_holidays_from_country(self.country, df)
+
+    def get_holidays_from_country(self, country: Union[str, Iterable[str], dict], df=None):
```

Comment:
`self` does not appear to be accessed in this method. I think we can safely move it to `hdays_utils.py`.